pages/test_analytics/public_test_suites.md

```diff
@@ -0,0 +1,24 @@
+# Public test suites
+
+If you're working on an open-source project, and want the whole world to be able to see your test analytics, you can make your test suite public.
+
+
+Making a suite public provides read-only public/anonymous access to:
+
+- All test suite data
+- Run results
+- Tests
```

Comment:
This might read as though the test code is public, when actually it's the analytics?